### ISSUE TITLE
Samples Import with Template - Error Name field

### ISSUE THREAD

**Issue by ttir** [c3g/fms#56]
Error: Name - This   field cannot be blank
--
There is no field in the template called   "Name"

It is the Individual Name. 
We do not   always have this information. 
Is there a default value to put? (Same as   sample name?)

[Sample_submission_Test_01_IndName.xlsx](https://github.com/c3g/fms/files/4523082/Sample_submission_Test_01_IndName.xlsx)

![Sample_Submission_ImportError01](https://user-images.githubusercontent.com/64210364/80109911-1a17d680-854c-11ea-9a13-c57d1669f0ee.jpg)

**Comment by abelisle789** [comment]
Yes, if we don't have the information, we can put the sample name. However, I'm agree that the error message should be more clear.

**Comment by dbujold** [comment]
@zxenia What was the message changed to?

**Comment by zxenia** [comment]
Same as #57 - it should get the template column name for the error message.

**Comment by dbujold** [comment]
Moving to "Future".
